app.js

```diff
@@ -0,0 +1,59 @@
+const option = document.querySelector("option");
+const main = document.querySelector("main");
+const button = document.querySelector("button");
+const allTitles = document.querySelector("#all-movie-titles"); //select tag
+const h3 = document.querySelector("#title");
+const ptag = document.querySelector("#release-date");
+const p = document.querySelector("#description");
+const form = document.querySelector("form");
+
+// const displayInfo = document.querySelector("#display-info");
+
+//get all movie titles getAlltitles gets movies from API
+const getAllTitles = async () => {
+  try {
+    const res = await axios.get("https://ghibliapi.herokuapp.com/films");
+    //gets data from API on films
+    res.data.forEach((movie) => {
+      const h3 = document.createElement("h3"); //gives title of movie
+      h3.innerText = movie.title;
+
+      const p = document.createElement("p"); //gives description
+      p.innerText = movie.description;
+      //   debugger;
+      const ptag = document.createElement("p"); //gives year
+      ptag.innerText = movie.release_date;
+
+      //   const displayInfo = document.querySelector("#display-info");
+      //   displayInfo.appendChild(h3); //title
+      //   displayInfo.appendChild(p); //description
+      //   displayInfo.appendChild(ptag); //year
+    });
+  } catch (err) {
+    console.log(err);
+  }
+};
+getAllTitles();
+
+const select = document.querySelector("select");
+const createOptions = (movie) => {
+  for (let i = 1; i <= movie; i++) {
+    const option = document.createElement("option");
+    option.innerHTML = movie.title;
+    option.value = i;
+    select.appendChild(option);
+  }
+};
```

Comment:
This should of been part of the getAllTitles. Instead of looping through the movie object you should been looping through the array of data that was returned from axios `res.data.forEach(movie => ...`